src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -362,7 +358,11 @@ namespace DataAirflowNetwork {
         }
     };
 
-    struct MultizoneCompDetOpeningProp // Large detailed opening component
+    struct AirflowElement
```

Comment:
At some point, this base class will presumably grow to include (at least) a pure virtual `calculate()` method, right?

Reply:
Yes, it didn't quite make into this PR.